src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/RejectApplicationsPostRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.Encoding;
+using SFA.DAS.LevyTransferMatching.Web.Attributes;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
+{
+    public class RejectApplicationsPostRequest
+    {
+        public string EncodedAccountId { get; set; }
+
+        [AutoDecode(nameof(EncodedAccountId), EncodingType.AccountId)]
+        public int AccountId { get; set; }
+
+        public string EncodedPledgeId { get; set; }
+
+        [AutoDecode(nameof(EncodedPledgeId), EncodingType.PledgeId)]
+        public int PledgeId { get; set; }
+
+        public List<int> ApplicationsToReject { get; set; }
```

Comment:
We should not expose internal, numeric Ids of objects in the UI. These should be encoded here and decoded again.

Reply:
I have removed the numeric IDs and now using the encodedApplicationIds. I have converted this array to a string array and querystring has now got the encoded application ids rather than numeric ids.